src/status_im/common/signals/events.cljs

```diff
@@ -108,6 +108,9 @@
       {:fx [[:dispatch [:profile.login/login-node-signal (transforms/js->clj event-js)]]]}
 
       "backup.performed"
-      {:db (assoc-in db [:profile/profile :last-backup] (oops/oget event-js :lastBackup))}
+      ;; Delay to improve the UX, if no delay is added, messages appear/disappear too fast.
```

Comment:
could you pls elaborate more what's the problem in details

Reply:
We have a message in the UI saying "Performing backup" but the signal returns too fast, so we dispatch the success event a little delayed so the UX is better, if it happens too fast the user isn't able to read the message.

Reply:
shouldn't this delay be in the component instead? because it might happen in other cases as well

Reply:
Going slightly off-topic: interestingly, again what we sometimes need in re-frame is to have a cleaner control of a priority queue. When this `backup.performed` signal arrives, it's not important to process it immediately and interrupt other important computations. There could be an effect named like:

```clojure
:fx [[:dispatch-x {:priority :low :dispatch event-vector ...
```